include/fmt/core.h

```diff
@@ -1083,11 +1095,23 @@ FMT_CONSTEXPR auto visit_format_arg(Visitor&& vis,
   case internal::type::char_type:
     return vis(arg.value_.char_value);
   case internal::type::float_type:
+#if FMT_USE_FLOAT
     return vis(arg.value_.float_value);
+#else
+    break;
+#endif
   case internal::type::double_type:
+#if FMT_USE_DOUBLE
     return vis(arg.value_.double_value);
+#else
+    break;
+#endif
   case internal::type::long_double_type:
+#if FMT_USE_LONG_DOUBLE
     return vis(arg.value_.long_double_value);
+#else
+    break;
+#endif
```

Comment:
Please remove conditional compilation here. `FMT_USE_*` should be handled on the visitor side instead.

Reply:
Makes sense, done.